feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewUiState.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.jetpackcamera.feature.preview
+
+
+/**
+ * Defines the current state of the [PreviewScreen].
+ */
+data class PreviewUiState(
+    val cameraState: CameraState = CameraState.NOT_READY,
+    val lensFacing: Int = 0, // CameraSelector.LENS_FACING_FRONT
```

Comment:
is the comment necessary? should it actually be `CameraSelector.LENS_FACING_FRONT`